api/comments/views.py

```diff
@@ -79,6 +79,7 @@ class CommentDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, Comm
         is_abuse       boolean            has this comment been reported by the current user?
         has_children   boolean            does this comment have replies?
         can_edit       boolean            can the current user edit this comment?
+        new_mentions   list               guids of mentioned users in content of comment
```

Comment:
> the html needs to be converted to markdown prior to saving

Gotcha. I'm seeing now where we convert [from markdown](https://github.com/CenterForOpenScience/osf.io/blob/eac466dba48543c997dbcbecc4ed2955e41ae23e/website/static/js/comment.js#L528) on load, and [to markdown](https://github.com/CenterForOpenScience/osf.io/blob/eac466dba48543c997dbcbecc4ed2955e41ae23e/website/static/js/comment.js#L280) on save. Hmm ...

> However, there has been discussion of removing the html bleaching from the API which would reduce the complexity of the conversions in comment.js (which are not ideal).

But we're not blocking this on that, are we?